### DISCUSSION TITLE
Can;t remove database

### DISCUSSION THREAD

**Discussion by callmeescape** [pawelsalawa/sqlitestudio#4857] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by callmeescape** [comment]
I had previously added a database to my sqlite studio, removed it, then tried to add it again, but it said the database was already there and there were no alternatives to open it.

**Comment by pawelsalawa** [comment]
Make sure you have this option checked in the menu and you see the databases list panel (here on the left side):
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/2249e6ff-9db9-41d9-80a5-1fd668175642)

There you will see all databases that are already added and you can remove from SQLiteStudio (but it will not delete actual files from the disk).
